workflow-templates/keyfactor-starter-workflow.yml

```diff
@@ -12,6 +12,8 @@ jobs:
       release_version: ${{ needs.call-create-github-release-workflow.outputs.release_version }}
       release_url: ${{ needs.call-create-github-release-workflow.outputs.release_url }}
       release_dir: EXAMPLE_SOLUTION/bin/Release/BUILD_TARGET # TODO: set build output directory to upload as a release, relative to checkout workspace
+    secrets: 
+      token: ${{ secrets.PRIVATE_REPO_ACCESS  }}

```

Comment:
probably not a big deal, but the token name is misleading since we are using it to access private packages, not repos

Reply:
I have updated the org level secret name as well as my PAT name